### PR TITLE
[Docs] Update windows installation guide for Meshery

### LINKED ISSUE #1395: [Docs] Fix missing documentation, logo path, Images and Table of Content
**Current State:**
- Documentation missing in https://meshery.layer5.io/docs/installation/platforms/windows
- The logos are broken in https://meshery.layer5.io/docs/installation/platforms
- Images are not shown in https://meshery.layer5.io/docs/guides/interpreting-performance-test-results
- Sections in the table of contents are repeated twice on,
   - https://meshery.layer5.io/docs/guides
   - https://meshery.layer5.io/docs/functionality

**Desired State:**
- [ ] Windows documentation should be added as soon as possible.
- [x] Logos should appear correctly.
- [x] Images should appear correctly.
- [ ] Table of contents should be fixed in,
  - https://meshery.layer5.io/docs/guides
  - https://meshery.layer5.io/docs/functionality

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by chandrashritii**
Added updated instructions for setting up Meshery on Windows

Signed-off-by: Shriti <shritichan@gmail.com>

**Description**

This PR fixes #1395. Added updated instructions on setting up and running Meshery on Windows

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by chandrashritii** [comment]
Adding a screenshot for reviewers:

Screenshot_2020-10-06 Windows

File: docs/pages/installation/windows.md [hunk 1]
```diff
@@ -1,48 +1,126 @@
...
+It is recommended to update to <a href="#wsl2">WSL2</a> as WSL1 doesn't support the Docker Desktop application for Windows. Instead, it only supports the deprecated version, [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/). 
 
-# Advanced Installation
+If you still wish to continue, follow the instructions for <button onclick="HideToggleFunction()"><b>WSL1</b></button>
 
-Users can control the specific container image and tag (version) of Meshery that they would like to run by editing their local `~/.meshery/meshery.yaml` (a docker compose file).
-Aligned with the Meshery container image, instead of leaving the implicit `:stable-latest` tag behind image: layer5/meshery, users will instead identify a specific image tag like so:
+<div id="hiddendiv">
+<p>
+1. The default versio of WSL is set to WSL1 by default. You can move forward to <a href="https://docs.microsoft.com/en-us/windows/wsl/install-win10#install-your-linux-distribution-of-choice">install the distro</a> of your choice. <br /><br />
```
Comment (nunix) [inline docs/pages/installation/windows.md:94]: Typo: The default versio**n**

**Comment by leecalcote** [comment]
@ameyaagashe will you kindly give this a once-over?

File: docs/pages/installation/windows.md [hunk 2]
```diff
@@ -1,48 +1,126 @@
...
+Set the default version to `WSL2`, which will be inherited by any distro you wish to use.
 
 ```bash
-./mesheryctl system start
+wsl --set-default-version 2
 ```
 
-#### Scoop
+#### 2. <b>[Install a new distro](https://docs.microsoft.com/en-us/windows/wsl/install-win10#install-your-linux-distribution-of-choice)</b>
+In this tutorial, [Ubuntu 18.04](https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q?activetab=pivot:overviewtab) will be the distro used. Feel free to use any distro of your choice.
```
Comment (ameyaagashe) [inline docs/pages/installation/windows.md:61]: We have given the URL, then the same or partial content is again repeated in the markdown file. 
I would point to the WSL URL(Microsoft page) and omit all until Step 3 Enable Docker.

**Review by ameyaagashe — COMMENTED** [review]
Comments added. Kindly check.

File: docs/pages/installation/windows.md (hunk 2 above)
Comment (chandrashritii) [reply docs/pages/installation/windows.md:61]: Thanks for the review @ameyaagashe Let me explain why the headings were added in order: 

- For the command `dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart Restart-Computer`, the tutorial is written in such a way that ideally, you wouldn't need to refer to the WSL documentation. The link is there for reference.
- For the WSL2 heading, we are recommending WSL2 over WSL1, which is why the Windows documentation was updated. Won't make sense if we remove that.
- The official documentation of WSL doesn't actually point to any particular distros and as we are working with Ubuntu 18.04 here throughout the tutorial, it is worth mentioning, in my opinion. Will edit if you and other reviewers disagree though.

File: docs/assets/js/main.js [hunk 3]
```diff
@@ -58,3 +58,14 @@ layout: null
     };
     Search.init();
 }(jQuery));
+
+function HideToggleFunction() {
+    var y = document.getElementById("hiddendiv");
```
[the remark this answers is not in the document]
Comment (chandrashritii) [reply docs/assets/js/main.js:63]: This is an internal code snippet. I'm not sure the variable name matters here so much but sure, I'll update it to a more meaningful name. Thanks for pointing this out @ameyaagashe

[the remark this answers is not in the document]
Comment (ameyaagashe) [reply docs/assets/js/main.js:63]: All good then from my side.

File: docs/pages/installation/windows.md (hunk 2 above)
Comment (chandrashritii) [reply docs/pages/installation/windows.md:61]: @ameyaagashe Could please explain which Powershell command you are referring to?
About checking their build version, we could add a short explanation on how to do that or refer them to a freeware software like [speccy](https://www.ccleaner.com/speccy) (Link is just for reference. Not the official site).
//CC @nunix

**Comment by ameyaagashe** [comment]
@chandrashritii 
To check the Windows Build version, the below PowerShell command could be of use.
`[System.Environment]::OSVersion.Version`

Another PowerShell command is below, extract from WSL Link:
**Step 3- Enable Virtual Machine Feature.**
`dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart`

**Comment by leecalcote** [comment]
@chandrashritii given the updates, is it time to re-request review on this Windows documentation?

**Comment by ameyaagashe** [comment]
Looking fine to me @chandrashritii

**Review by leecalcote — APPROVED** [review]

**Pull request merged**